src/Compilers/CompilersPackage/source.extension.vsixmanifest

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="CompilersPackage.Roslyn.c00e9a7d-047b-4152-8cdf-8af9c14bd087" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>CompilersPackage</DisplayName>
+    <Description xml:space="preserve">The project-willow roslyn compilers package.</Description>
```

Comment:
Is this string viewable by customers? "Roslyn" and "Willow" should be uppercase, minimally.